hack/install/config.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-export OSS_ENDPOINT="oss-cn-beijing.aliyuncs.com"
-export OSS_BUCKET="pouch-oss"
+export OSS_ENDPOINT="oss-cn-hongkong.aliyuncs.com"
+export OSS_BUCKET="pouch-oss-hk"
```

Comment:
how to add this sign?

Reply:
I don't know. 🐶

Reply:
I like it but I don't know to do that. LoL